pkgs/os-specific/bsd/freebsd/pkgs/mount.nix

```diff
@@ -1,13 +1,24 @@
 {
   mkDerivation,
+  lib,
   libutil,
   libxo,
-  ...
+  libkiconv,
+  withMsdosfs ? true,
 }:
 mkDerivation {
   path = "sbin/mount";
+  extraPaths = lib.optionals withMsdosfs [ "sbin/mount_msdosfs" ];
   buildInputs = [
     libutil
     libxo
-  ];
+  ] ++ lib.optionals withMsdosfs [ libkiconv ];
+
+  postBuild = lib.optionalString withMsdosfs ''
+    make -C $BSDSRCDIR/sbin/mount_msdosfs $makeFlags
+  '';
+
+  postInstall = lib.optionalString withMsdosfs ''
+    make -C $BSDSRCDIR/sbin/mount_msdosfs $makeFlags install
+  '';
```

Comment:
Can these no be separate?

Reply:
They can be separate, but mount (like fsck) is kind of useless without its helper programs. Do you think it's actually worth separating them? We also have the option of patching it to look up the helper programs by embedded absolute path instead of by $PATH, which would make them an explicit dependency instead of an implicit dependency (old option) or a bundled dependency (current option)

Reply:
I basically just think its good to keep the existing packaging structure where possible. I don't have much opinion on explicit vs implicit dependency --- I image its not too bad to arrange for NixBSD to install all 3 if its just an implicit dependency.